ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_SERVERS.yml

```diff
@@ -389,6 +389,33 @@ servers:
           mode: active
           enabled: false
 
+  server18_monitoring_session_source:
+    rack: RackC
+    adapters:
+      - server_ports: [ Eth1 ]
+        switch_ports: [ Ethernet25 ]
+        switches: [ DC1-SVC3A ]
+        profile: TENANT_A
+        monitor_sessions:
```

Comment:
Either change this into a list, to support multiple monitor sessions (would be nice), or use singular form on the key name.